recipes/libcxxwrap-julia/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib $SRC_DIR
+make install
```

Comment:
Is there a `make check` that we can run?

Reply:
You can run tests using ctest, but this would be more to build and run as part of conda-forge's CI. I made some basic file presence tests in the meta.yaml to verify that things are installed properly.